modules/RTC/JitsiTrack.js

```diff
@@ -310,4 +312,46 @@ JitsiTrack.prototype.getMSID = function () {
     return (streamId && trackId) ? (streamId + " " + trackId) : null;
 };
 
+/**
+ * Set new audio output device for track's DOM elements.
+ * @param {string} audioOutputDeviceId - id of 'audiooutput' device from
+ *      navigator.mediaDevices.enumerateDevices()
+ * @emits JitsiTrackEvents.TRACK_AUDIO_OUTPUT_CHANGED
+ * @returns {Promise}
+ */
+JitsiTrack.prototype.setAudioOutput = function (audioOutputDeviceId) {
+    var self = this;
+
+    if (!RTCUtils.isAudioOutputDeviceChangeAvailable()) {
+        return Promise.reject(
+            new Error('Audio output device change is not supported'));
+    }
+
+    return Promise.all(this.containers.map(function(element) {
+        return element.setSinkId(audioOutputDeviceId)
+            .catch(function (error) {
+                console.error('Failed to change audio output device on element',
+                    element, error);
+
+                // TODO: for some reason 'AbortError' is raised on video
```

Comment:
Actually found the reason why this happens - when media stream from GUM has only video tracks and no audio tracks. In RTCUtils handleLocalStream function we actually split stream obtained from GUM into separate audio and video streams. So I suggest just to ignore elements for local tracks at all in this case. And maybe even all 'video' elements can be ignored as audio communication is done via 'audio' elements?